src/main/java/cloud/eppo/api/Configuration.java

```diff
@@ -115,10 +122,15 @@ public boolean isConfigObfuscated() {
     return isConfigObfuscated;
   }
 
+  @SuppressWarnings("unused")
+  public byte[] getFlagConfigJson() {
```

Comment:
I wonder if we should call this `getFlagCOnfigJsonBytes()`

Reply:
There is only one `getFlagConfigJson` method (so nothing to disambiguate) and the type is right there so I didn't think adding `Bytes` to the name would really add much. OTOH, extra description is rarely hurtful.